src/shared/seccomp-util.c

```diff
@@ -1112,6 +1112,17 @@ int seccomp_memory_deny_write_execute(void) {
                 r = seccomp_rule_add_exact(
                                 seccomp,
                                 SCMP_ACT_ERRNO(EPERM),
+                                SCMP_SYS(mmap2),
```

Comment:
I mean `s390` doesn't have `mmap2(1, 2, 3, 4, 5, 6)`
Because:
```
/*
 * Perform the mmap() system call. Linux for S/390 isn't able to handle more
 * than 5 system call parameters, so this system call uses a memory block
 * for parameter passing.
 */
```

 So, `glibc` should pack all args into struct and call `mmap2(&packed_mmap_args)`.
I didn't check. But `glibc`'s changelog contains:
```
ChangeLog.12:   * sysdeps/unix/sysv/linux/s390/mmap64.S: Use mmap if mmap2 isn't
ChangeLog.12:   * sysdeps/unix/sysv/linux/s390/mmap.S: Use mmap2 if it is present.
```
Does anybody check this?